kernel/main.c

```diff
@@ -340,6 +353,116 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services(void)
+{
+   struct mem_region ma;
+
+   if (!booting_with_uefi)
+      return;
+
+   EFI_RUNTIME_SERVICES *RT = TO_PTR(uefi_rt_addr);
```

Comment:
Move to this variable declaration on the top. You can assign it's value later. Again, I know that in C++ that would be an abomination.